openstack/cronus/templates/cronus/_config.yaml.tpl

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.cronus.enabled -}}
+cronus:
+  cacheSize: {{ .Values.cronus.cacheSize }}
+  billingCacheTTL: {{ .Values.config.billingCacheTTL }}
+  aws:
+    forwardUserAgent: {{ .Values.config.forwardUserAgent }}
+    allowedServices:
+      email: {{ .Values.config.allowedServices.email }}
+  listenAddr:
+    http: # default :5000
+    smtp: # default :1025
+  keystone:
+    authUrl: {{ .Values.config.authUrl }}
+    applicationCredentialID: {{ .Values.config.applicationCredentialID }}
+    applicationCredentialSecret: {{ .Values.config.applicationCredentialSecret }}
+    region: {{ .Values.config.region }}
+    endpointType: {{ .Values.config.endpointType }}
+    verifyEmailDomain: example.com
+    verifyEmailSecret: domainkey
```

Comment:
Can you parametrize above two values? Besides these values have to be placed under the `aws` in `openstack/cronus/templates/nebula/_config.yaml.tpl`.